context/app/static/js/components/detailPage/BulkDataTransfer/BulkDataTransfer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useAppContext } from 'js/components/Contexts';
+import SectionHeader from 'js/shared-styles/sections/SectionHeader';
+import { DetailPageSection } from 'js/components/detailPage/style';
+import NotLoggedIn from './NotLoggedIn';
+import LoggedIn from './LoggedIn';
+
+function BulkDataTransfer() {
+  const { isAuthenticated } = useAppContext();
+  return (
+    <DetailPageSection id="bulkDataTransfer">
```

Comment:
```suggestion
    <DetailPageSection id="bulk-data-transfer">
```
Use kebab casing for ids.

The id needs to reflect what's used for the table of contents on the dataset page.